packages/checkbox/test/checkbox.test.ts

```diff
@@ -174,6 +174,7 @@ describe('Checkbox', () => {
         HTMLElement.prototype.focus.apply(el);
         el.focusElement.dispatchEvent(new CustomEvent('focusout'));
         await nextFrame();
+        await nextFrame();
```

Comment:
Was this checkbox test not passing before without the additional nextFrame?

Reply:
It started being flaky, like maybe 1 in 20 failures... 😞